.gitlab/package_build/deb.yml

```diff
@@ -1,71 +1,9 @@
 ---
```

Comment:
If I understand correctly, we might be able to rename the files, since `package_build` should no longer contain things related to `deb/rpm` but only `arm/amd` differences.
Or this one can probably be changed to `heroku.yml` ?